src/dmd/cppmangle.d

```diff
@@ -150,18 +150,21 @@ private final class CppMangleVisitor : Visitor
     Objects components;         // array of components available for substitution
     OutBuffer* buf;             // append the mangling to buf[]
     Loc loc;                    // location for use in error messages
+    Strings* abiTags;           // record of the processed abi-tags
 
     /**
      * Constructor
      *
      * Params:
      *   buf = `OutBuffer` to write the mangling to
      *   loc = `Loc` of the symbol being mangled
+     *   abiTags = output to store the processed abi-tags
      */
-    this(OutBuffer* buf, Loc loc)
+    this(OutBuffer* buf, Loc loc, Strings* abiTags = null)
```

Comment:
I know Walter doesn’t like default values for parameters.

Reply:
Why?

Reply:
I think it's something with being less readable, but you'll have to ask him.

Reply:
Well, pick your poison. Either this or an overload.

Reply:
I prefer a default value, but it’s not my decision.

It’s also possible to explicitly pass in the default value.